Sources/TSCBasic/Path.swift

```diff
@@ -467,24 +466,22 @@ extension Path {
     }
 }
 
-private struct UNIXPath: Path {
+#if os(Windows)
+private struct WindowsPath: Path {
     let string: String
 
-    static let root = UNIXPath(string: "/")
+    static let root = WindowsPath(string: "/")
```

Comment:
I think that this should be `\`.

```suggestion
    static let root = WindowsPath(string: "\\")
```
```suggestion
    static let root = WindowsPath(string: "/")
```

Reply:
I have not changed the logic here from the original code: https://github.com/apple/swift-tools-support-core/blob/main/Sources/TSCBasic/Path.swift#L473